lib/eChronicler_web/templates/journal_entry/index.html.heex

```diff
@@ -0,0 +1,15 @@
+<h1 id="title">Journal Entries Page</h1>
+
+<div>
+ 
+    <%= for journal_entry <- @journal_entries do %>
+      <div id="entry">
+        <h3 id="title">Title: <%= journal_entry.title %></h3>
+        Author: <%= journal_entry.author %> <br>
+        Published: <%= journal_entry |> EChronicler.EChronicler.JournalEntry.format_datetime() %> <br>
+        Entry: <%= journal_entry.entry |> EChronicler.EChronicler.JournalEntry.truncate_journal_entry_text() %>
```

Comment:
Is there a migration to support the longer journal entries in the DB?  Thought I saw something on slack about that...

Reply:
Done!